devices/sinope.js

```diff
@@ -44,36 +49,21 @@ module.exports = [
             await reporting.thermostatTemperature(endpoint);
             await reporting.thermostatPIHeatingDemand(endpoint);
             await reporting.thermostatOccupiedHeatingSetpoint(endpoint);
-
-            try {
-                await reporting.thermostatSystemMode(endpoint);
-            } catch (error) {/* Not all support this */}
-
-            try {
-                await reporting.thermostatRunningState(endpoint);
-            } catch (error) {/* Not all support this */}
+            await reporting.thermostatSystemMode(endpoint);
```

Comment:
I don't think the try/catch should be removed as not all thermostats seem to support this and this causes the configure to fail.

Reply:
Set back both try catches for both thermostats this time. 

Running state seems to be handle by piheatingdemand (if more than 10 = running). 
I set it back anyways